priv/repo/migrations/20160707161837_add_revoked_to_keys.exs

```diff
@@ -0,0 +1,11 @@
+defmodule HexWeb.Repo.Migrations.AddRevokedToKeys do
+  use Ecto.Migration
+
+  def change do
+    alter table(:keys) do
+      add :revoked_name, :text
```

Comment:
Since `:name` has a unique constraint, it allows new keys to be created that use the same name as old ones.